### ISSUE TITLE
Softlocked with home button

### ISSUE THREAD

**Issue by alispostkiste** [JonasBernard/FakeStandby#19]
I have a Galaxy S8+, so whenever turn off the screen via Fake Standby and then push the "Home Button" the screen turns on but only the half of it. The other half stays black. Also the black half does not react to any input and when i turn off the screen it does not react to the 4 finger unlock gesture. The only way to resolve this, by now, is to force a restart of the phone via a button combination.

If i can provide you more information let me know

**Comment by JonasBernard** [comment]
Okay that sounds weird. I don't know, how you can help me with that. Are you using the newest version available on F-Droid?

**Comment by alispostkiste** [comment]
Yes its the newest version. I think pushing the home button triggers the lockscreen, which puts it in a weird state... but that is just guessing

**Comment by JonasBernard** [comment]
Mh. Is this process "animated"? Or does it get there immediately?

**Comment by userdaphi** [comment]
That sounds for me like the screenrotation bug:
same like #8 & #10 I think

I was able to reproduce it I think. This specially happens when the phone does a screenrotation in FakeStandby mode. When the phone gots turned back to vertical the 3/4 of the screen is black and the rest is normal. But touchscreen does not react on any interaction. I always have to power off the phone and reboot.
But like I told you, it only happens when screenrotation is activated. Turn screenrotaton off and you're fine.
